functions/Get-DbaPageFileSetting.ps1

```diff
@@ -1,159 +1,108 @@
-   This function needs to be executed as a user with local admin rights on the target computer(s).
-
-.PARAMETER ComputerName
-The Server that you're connecting to.
-This can be the name of a computer, a SMO object, an IP address or a SQL Instance.
-
-.PARAMETER Credential
-Credential object used to connect to the Computer as a different user
+#ValidationTags#Messaging,FlowControl,Pipeline,CodeStyle#
 
-.PARAMETER EnableException
+function Get-DbaPageFileSetting {
+<#
+    .SYNOPSIS
+        Returns information on the pagefile configuration of the target computer.
+    
+    .DESCRIPTION
+        This command uses CIM (or other, related computer management tools) to detect the pagefile configuration of the target compuer(s).
+        Note that this may require local administrator privileges for the relevant computers.
+    
+    .PARAMETER ComputerName
+        The Server that you're connecting to.
+        This can be the name of a computer, a SMO object, an IP address, an AD COmputer object, a connection string or a SQL Instance.
+    
+    .PARAMETER Credential
+        Credential object used to connect to the Computer as a different user
+    
+    .PARAMETER EnableException
         By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
         This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
         Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
-
-.NOTES
-Tags: CIM
-Author: Klaas Vandenberghe ( @PowerDBAKlaas )
-
-dbatools PowerShell module (https://dbatools.io)
-Copyright (C) 2016 Chrissy LeMaire
-License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
-
-.LINK
- https://dbatools.io/Get-DbaPageFileSetting
-
-.EXAMPLE
-Get-DbaPageFileSetting -ComputerName ServerA,ServerB
-
-Returns a custom object displaying ComputerName, AutoPageFile, FileName, Status, LastModified, LastAccessed, AllocatedBaseSize, InitialSize, MaximumSize, PeakUsage, CurrentUsage  for ServerA and ServerB
-
-.EXAMPLE
-'ServerA' | Get-DbaPageFileSetting
-
-Returns a custom object displaying ComputerName, AutoPageFile, FileName, Status, LastModified, LastAccessed, AllocatedBaseSize, InitialSize, MaximumSize, PeakUsage, CurrentUsage  for ServerA
-
+    
+    .EXAMPLE
+        Get-DbaPageFileSetting -ComputerName ServerA,ServerB
+        
+        Returns a custom object displaying ComputerName, AutoPageFile, FileName, Status, LastModified, LastAccessed, AllocatedBaseSize, InitialSize, MaximumSize, PeakUsage, CurrentUsage  for ServerA and ServerB
+    
+    .EXAMPLE
+        'ServerA' | Get-DbaPageFileSetting
+        
+        Returns a custom object displaying ComputerName, AutoPageFile, FileName, Status, LastModified, LastAccessed, AllocatedBaseSize, InitialSize, MaximumSize, PeakUsage, CurrentUsage  for ServerA
+    
+    .NOTES
```

Comment:
Just reference the notes and links are put above the examples in almost all of the commands. Not sure if your template in PSS may reformat it this way?